docs/book/src/reference/reference.md

```diff
@@ -5,6 +5,9 @@
     Finalizers are a mechanism to
     execute any custom logic related to a resource before it gets deleted from
     Kubernetes cluster.
+  - [Watching Objects](watching-objects.md)
+    Controller Runtime provides the ability to watch objects relevant to
+    the objects that you own.
   - [Kind cluster](kind.md)
```

Comment:
The watch feature allows you to observe relevant resources for your controller not only those ones that are onwed by the controller.